packages/render-fidelity-tools/src/components/renderers/dspbr-pt-viewer.ts

```diff
@@ -80,29 +80,29 @@ export class PathtracingViewer extends LitElement {
     this[$renderer] = new PathtracingRenderer(this[$canvas]!, enableControls);
 
     const renderer = this[$renderer];
-    renderer.autoScaleOnImport(false);
     renderer.setPixelRatio(0.5);
-    renderer.setMaxBounceDepth(8);
+    renderer.setMaxBounceDepth(4);
```

Comment:
Due to the changes in the path termination logic, the renderer should now be able to produce reasonable results with shorter path lengths. For the current set of scenarios a value of 4 should be sufficient.